docker/labels.go

```diff
@@ -11,10 +11,11 @@ type Label string
 
 // Libcompose default labels.
 const (
-	NAME    = Label("io.docker.compose.name")
-	PROJECT = Label("io.docker.compose.project")
-	SERVICE = Label("io.docker.compose.service")
-	HASH    = Label("io.docker.compose.config-hash")
+	NAME     = Label("io.docker.compose.name")
+	PROJECT  = Label("io.docker.compose.project")
+	SERVICE  = Label("io.docker.compose.service")
+	HASH     = Label("io.docker.compose.config-hash")
+	RECREATE = Label("io.docker.compose.recreate")
```

Comment:
I think a better way to implement it would making the `forceRecreate` param we talk about here: https://github.com/docker/libcompose/pull/89/files#r43012067